frontend/src/utils/serializeFilterParams.js

```diff
@@ -38,11 +38,19 @@ export default function serializeFilterParams(filters, descriptions) {
 
       case FILTER_TYPE.INPUT: {
         const options = filters[field].options
-        key = (options && options.exactMatch) ? (key + "__startswith") : (key + "__icontains")
+
+        if (value.includes(',')) {
+          key += "__in"
+        } else if (options?.exactMatch) {
+          key += "__startswith"
```

Comment:
We need at some point down the line to have an exact match that actually do what is advertised ... eventually people will complain about that (e.g. i want sample juice_1 and i keep getting juice_12 too!).

Reply:
That was discussed previously in the implementation of the exact match with Romain. Apparently those kind of 'exact matches' are rather starts_with than exact =.
The feature was approved, and to be honest I wouldnt replace it with a '=' neither... If you strongly disagree we can discuss this in private, or at a team meeting.

Reply:
I can understand your point, but we shouldnt make a decision here :P let's bring it back to a team discussion (with David)